predict-next-purchase/Tutorial.ipynb

```diff
@@ -482,44 +642,44 @@
    "source": [
     "## Step 2: Create a Featuretools entityset\n",
     "\n",
-    "When using Dask dataframes to create an entityset, variable type inference is not performed as it is with entitysets created from pandas dataframes. As a result, users must specify the Featuretools variable types for all of the columns in the dataframes that make up the entityset when using Dask. In the following cell we define the data types for the `order_products` and `orders` entities."
+    "When using Dask dataframes to create an entityset, type inference can be computationally intensive compared to entitysets created from pandas dataframes because of needing to pull the distributed data into memory. As a best practice, users should specify the Woodwork logical types for all of the columns in the dataframes that make up the entityset when using Dask. In the following cell we define the data types for the `order_products` and `orders` dataframes."
```

Comment:
This would be a good place to reference the woodwork logical types guide--especially now that we're just listing them as strings instead of objects - https://woodwork.alteryx.com/en/stable/guides/logical_types_and_semantic_tags.html#Logical-Types